martian/src/metadata.rs

```diff
@@ -59,6 +61,22 @@ impl Default for Version {
     }
 }
 
+#[derive(Debug, Clone, Serialize, Deserialize)]
+#[serde(rename_all = "snake_case")]
+pub enum ProfileMode {
```

Comment:
```suggestion
#[non_exhaustive]
pub enum ProfileMode {
```
It's legal for people to configure arbitrary additional modes.